### PR TITLE
examples/Readme.md: Update description to be consistent with the folder names

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

**Comment by rht** [comment]
Partially addresses #693, where `bank_reserves` is added to the `examples/Readme.md`.

File: examples/Readme.md [hunk 1]
```diff
@@ -1,38 +1,57 @@
...
 This directory contains example models meant to test and demonstrate Mesa's features, and provide demonstrations for how to build and analyze agent-based models. For more information on each model, see its own Readme and documentation.
 
+## Models
+
+Classic models, some of which can be found in NetLogo's/MASON's example models.
+
+### bank_reserves
+A highly abstracted, simplified model of an economy, with only one type of agent and a single bank representing all banks in an economy.
+
 ### Color Patches
```
Comment (Corvince) [inline examples/Readme.md]: should be color_patches

File: examples/Readme.md [hunk 2]
```diff
@@ -1,38 +1,57 @@
...
 [Sugarscape 2 Constant Growback](http://ccl.northwestern.edu/netlogo/models/Sugarscape2ConstantGrowback)
+
+### virus_on_network
+This model is based on the NetLogo model "Virus on Network".
+
+## Examples
+
+Example models specifically for demonstrating Mesa's features.
+
+## charts
```
Comment (Corvince) [inline examples/Readme.md]: should be ### charts (one more #)

File: examples/Readme.md [hunk 3]
```diff
@@ -1,38 +1,57 @@
...
 
-### Sugarscape CG
+### sugarscape_cg
 Implementation of Sugarscape 2 Constant Growback model, based on the Netlogo
 [Sugarscape 2 Constant Growback](http://ccl.northwestern.edu/netlogo/models/Sugarscape2ConstantGrowback)
+
+### virus_on_network
+This model is based on the NetLogo model "Virus on Network".
+
+## Examples
```
Comment (Corvince) [inline examples/Readme.md]: I think "examples" is too generic (the whole folder is called examples). Maybe "Feature examples"?

File: examples/Readme.md [hunk 4]
```diff
@@ -1,38 +1,57 @@
...
+### boid_flockers
 [Boids](https://en.wikipedia.org/wiki/Boids)-style flocking model, demonstrating the use of agents moving through a continuous space following direction vectors.
 
-### ForestFire
+### forest_fire
 Simple cellular automata of a fire spreading through a forest of cells on a grid, based on the NetLogo [Fire model](http://ccl.northwestern.edu/netlogo/models/Fire).
 
-### PD_Grid
+### hex_snowflake
+Conway's game of life on a hexagonal grid. TODO example/hex_snowflake/Readme.md needs to be updated.
```
Comment (Corvince) [inline examples/Readme.md]: I think we should resolve this TODO within this PR or have an issue or other PR ready. I think this readme file is too visible for users to include TODO texts (which are for contributers and not users)

**Comment by rht** [comment]
I updated the `hex_snowflake` based on my reading of the code and #409.
The initialization
```
        # Place a dead cell at each location.
        # activate the center(ish) cell.
```
is slightly different from 
```
        # Place a cell at each location, with some initialized to
        # ALIVE and some to DEAD.
```
of `conways_game_of_life`. I'm not entirely sure if the initialization doesn't affect the essence of the result, which I think it doesn't. cc @JamesHovet

**Comment by rht** [comment]
The last commit shouldn't unblock the first 2 commits. I'm making another PR for the last one.

**Comment by Corvince** [comment]
Thanks for the updates, looks good to me!

**Pull request merged**